utility/importer/chrome_importer.cc

```diff
@@ -108,6 +108,27 @@ bool SetEncryptionKeyForPasswordImporting(
 }
 #endif
 
+class CreateCopyFile {
```

Comment:
I believe ScopedTempFile is what you want here

Reply:
Do you want copying `ScopedTempFile` to here and use it in this file?
Unfortunately, `ScopedTempFile` could be only used by chromecast module's testcase.

Reply:
I didn't notice that, but then this should be `ScopedCopyFile`